snowpack/src/build/process.ts

```diff
@@ -147,18 +147,13 @@ export function maybeCleanBuildDirectory(state: BuildState) {
 
 export async function addBuildFiles(state: BuildState, files: string[]) {
   const {config} = state;
-
-  const excludePrivate = new RegExp(`\\${path.sep}\\..+(?!\\${path.sep})`);
```

Comment:
note; I tested this regex in regex101 and it wasn't behaving as I believe it was intended to be written (`.+` would swallow everything, so that `(?!\\${path.sep}` was always ignored).

The other `excludePrivate` regex didn't include this handling, so they basically both functioned the same way.

I've combined them both into a single Regex in `util.ts`